sql/catalyst/src/main/scala/org/apache/spark/sql/connector/expressions/expressions.scala

```diff
@@ -112,15 +112,9 @@ private[sql] final case class BucketTransform(
     arguments.collect { case named: NamedReference => named }
   }
 
-  override def arguments: Array[Expression] = numBuckets +: columns.toArray
+  override def arguments: Array[Expression] = (numBuckets +: columns.toArray) ++ sortedColumns
```

Comment:
does this really work? I don't see a way for people to extract bucket and sort columns from arguments with the `Transform` API.